outscale/data_source_outscale_net.go

```diff
@@ -56,15 +54,9 @@ func dataSourceOutscaleOAPIVpc() *schema.Resource {
 
 func dataSourceOutscaleOAPIVpcRead(d *schema.ResourceData, meta interface{}) error {
 	conn := meta.(*OutscaleClient).OSCAPI
-
 	req := oscgo.ReadNetsRequest{}
-
-	if v, ok := d.GetOk("filter"); ok {
-		req.SetFilters(buildOutscaleOAPIDataSourceNetFilters(v.(*schema.Set)))
-	}
-
-	if id := d.Get("net_id"); id != "" {
-		req.Filters.SetNetIds([]string{id.(string)})
+	if filters, filtersOk := d.GetOk("filter"); filtersOk {
+		req.SetFilters(buildOutscaleOAPIDataSourceNetFilters(filters.(*schema.Set)))
 	}
```

Comment:
This is a singular data_source, therefore to avoid matching with multiple data_source user should add
```
filtersilters, filtersOk := d.GetOk("filter")
if !filtersOk {
   return fmt.Errorf("filters must be assigned")
}
```